examples/src/index.js

```diff
@@ -140,7 +142,7 @@ function Demo() {
             />
           </Grid.Column>
         </Grid>
-        <Divider vertical>Preview</Divider>
+        <Divider vertical></Divider>
```

Comment:
This can be self-closing (`<Divider vertical />`

Reply:
Yes, made the change. Thanks :+1: